pointcloud/merging_test.go

```diff
@@ -12,8 +12,6 @@ import (
 
 	"go.viam.com/rdk/spatialmath"
 )
-
-//nolint:unused
```

Comment:
[minor] Deal with linter removing this and the protoutils  nolint:staticcheck later.